frontend/src/modules/automation/components/webhooks/webhook-form.vue

```diff
@@ -232,15 +234,18 @@ import { i18n } from '@/i18n'
 import integrationsJson from '@/jsons/integrations.json'
 import activityTypesJson from '@/jsons/activity-types.json'
 import UrlField from '@/shared/fields/url-field'
-import _ from 'lodash'
+import isEqual from 'lodash/isEqual'
```

Comment:
Thanks for reminding me about this, I might need to do this change in other places